### DISCUSSION TITLE
I want an overhaul of the Date object

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#215] [Wants]
The `Date` object is not as robust and intuitive as it should be. Let’s take a page from libraries like [Moment.js](https://momentjs.com/) and improve `Date`.

<hr>

https://webwewant.fyi/wants/128/

**Comment by aciccarello** [comment]
There's currently a JavaScript proposal for that called [Temporal](https://github.com/tc39/proposal-temporal)
